doc/source/examples/cfd-dem/square-fluidized-bed/square-fluidized-bed.rst

```diff
@@ -0,0 +1,390 @@
+==================================
+Gas-Solid Fluidized Bed
+==================================
+
+It is strongly recommended to visit `CFD-DEM parameters <../../../parameters/unresolved_cfd-dem/unresolved_cfd-dem.html>`_ for more detailed information on the concepts and physical meaning of the parameters CFD-DEM.
+
+Features
+----------------------------------
+- Solvers: ``cfd_dem_coupling_3d``
+- Three-dimensional problem
+- Displays the selection of models and physical properties.
+- Simulates aa solid-gas fluidized bed.
+
+
+Location of the examples
+------------------------
+ ``/examples/cfd-dem/square_fluidized_bed``
+
+
+Description of the case
+-----------------------
+
+This example simulates the fluidization of spherical particles in air. First, we use Lethe-DEM to fill the bed with particles. Then, we use the ``cfd_dem_coupling`` solver within Lethe to simulate the fluidization of the particles.
+
+
+Parameter file
+--------------
+
+Lethe simulations are controlled by *parameter files* which possess the extension ``.prm``. This is the default text format of the ParameterHandler class of the deal.ii library from which Lethe derives. For more information on this class, we refer to the `deal.II documentation <https://www.dealii.org/current/doxygen/deal.II/classParameterHandler.html>`_. 
+
+Parameter files are made of subsections which describe a portion of the simulation (e.g. physical properties, simulation control). In parameter files, lines starting with ``#`` are comments. Parameters are set using the following syntax:
+
+.. code-block:: text
+
+    set parameter name = value
+
+The syntax is flexible. Parameters do not need to be specified in a specific order, but only within the subsection in which they belong. All parameter subsections are described in the `parameter section <../../../parameters.html>`_ of the documentation.
+
+To set-up the square fluidized bed case, we first fill the bed with particles. 
+We first introduce the different sections of the parameter file (packing_in_cylinder.prm) needed to run this simulation. In this example, we are simulating a squred fluidized that has a half length of 0.1 m, and a side of 0.04 m. We use the GridGenerator::subdivided_hyper_rectangle function of Deal.II in order to generate the mesh. The square bed is divided 40 times in the y direction. The following portion of the DEM parameter file shows the function called:
```

Comment:
I think this part does not have to be copied from one example to another, as examples other than Lid-driven cavity don't have it.